Source/WebCore/page/scrolling/ScrollingCoordinator.cpp

```diff
@@ -126,7 +126,7 @@ EventTrackingRegions ScrollingCoordinator::absoluteEventTrackingRegionsForFrame(
     if (auto* scrollableAreas = frameView->scrollableAreas()) {
         for (auto& scrollableArea : *scrollableAreas) {
             // Composited scrollable areas can be scrolled off the main thread.
-            if (!scrollableArea->isVisibleToHitTesting() || scrollableArea->usesAsyncScrolling())
+            if (!scrollableArea->isVisibleToHitTesting() || scrollableArea->usesAsyncScrolling() || scrollableArea->needsAnimatedScroll())
```

Comment:
Isn't this still going to cause animated-scrolling ScrollableAreas to be missing from the non-fast event region while the animation is running, which means they won't get WheelEvents?

Reply:
Removed this extra condition since we now use another set to hold the non user scrollable areas.